Formula/o/ocicl.rb

```diff
@@ -0,0 +1,48 @@
+class Ocicl < Formula
+  desc "OCI-based ASDF system distribution and management tool for Common Lisp"
+  homepage "https://github.com/ocicl/ocicl"
+  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.3.4.tar.gz"
+  sha256 "be31c1d5de3352bf2ddc759153abc9c04ce4c0b962cc1d9d07888379e18ff202"
+  license "MIT"
+
+  depends_on "oras"
+  depends_on "sbcl"
+  depends_on "zstd"
+
+  def install
+    mkdir_p libexec
+    mkdir_p bin
+
+    system "sbcl", "--dynamic-space-size", "3072", "--no-userinit", "--eval",
+           "(require 'asdf)", "--eval", <<~LISP
+             (progn
+               (push (uiop:getcwd) asdf:*central-registry*)
+               (asdf:load-system :ocicl)
+               (sb-ext:save-lisp-and-die "#{libexec}/ocicl.core"))
+           LISP
```

Comment:
Can you add a comment explaining why we're not using the upstream-provided `setup.lisp` to build here?